app/src/main/java/org/fossasia/susi/ai/device/managedevices/ManageDeviceActivity.kt

```diff
@@ -0,0 +1,38 @@
+package org.fossasia.susi.ai.device.managedevices
+
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.view.MenuItem
+import org.fossasia.susi.ai.R
+
+class ManageDeviceActivity : AppCompatActivity() {
+
+    private val TAG_MUSIC_SERVERS_FRAGMNENT = "MusicServersFragment"
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        overridePendingTransition(R.anim.trans_left_in, R.anim.trans_left_out)
+        setContentView(R.layout.activity_manage_device)
+
+        val musicServerFragment = MusicServersFragment()
+        supportFragmentManager.beginTransaction()
+                .replace(R.id.manage_device_container, musicServerFragment, TAG_MUSIC_SERVERS_FRAGMNENT)
+                .commit()
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
```

Comment:
Then shift it to fragment

Reply:
Why so? This is the method that has been used in all other activities present in the app.